app/views/dashboard/_host_reports_widget.html.erb

```diff
@@ -0,0 +1,27 @@
+<h4 class="header">
+  <%= _("Host Latest Reports") %>
+</h4>
+<% if @data.latest_events? %>
```

Comment:
In plugin widgets you cannot load data into `@data` in app/services/dashboard/data.rb you have to do it in the ERB template. You can share the SQL queries as scopes or methods in either HostReport or Host::Managed models (the plugin has a concern class for extensions). Example: https://github.com/theforeman/foreman_discovery/blob/develop/app/views/dashboard/_discovery_widget.html.erb#L1